meteor-app/.deploy/mup.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  servers: {
+    one: {
+      // TODO: set host address, username, and authentication method
+      host: '178.128.211.31',
+      username: 'root',
+      // pem: './path/to/pem'
+      // password: 'server-password'
+      // or neither for authenticate from ssh-agent
+    }
+  },
+
+  app: {
+    // TODO: change app name and path
+    name: 'eryk-b2b',
+    path: '../',
+
+    servers: {
+      one: {},
+    },
+
+    buildOptions: {
+      serverOnly: true,
+    },
+
+    env: {
+      // TODO: Change to your app's url
+      // If you are using ssl, it needs to start with https://
+      ROOT_URL: 'http://178.128.211.31',
+      MONGO_URL: 'mongodb+srv://eryk:UnPIUIrpfbQlUGed@erykcluster-2bkuo.mongodb.net/bikes',
+    },
+
+    docker: {
+      // change to 'abernix/meteord:base' if your app is using Meteor 1.4 - 1.5
+      image: 'abernix/meteord:node-8.4.0-base',
+    },
+
+    // Show progress bar while uploading bundle to server
+    // You might need to disable it on CI servers
+    enableUploadProgressBar: true,
+    deployCheckWaitTime: 30
+  },
+
+
+  // (Optional)
+  // Use the proxy to setup ssl or to route requests to the correct
+  // app when there are several apps
+
+  // proxy: {
+  //   domains: 'mywebsite.com,www.mywebsite.com',
+
+  //   ssl: {
```

Comment:
You should enable this, so that https is enabled (don't worry, mup will do all the hard work for you. Then you can put the meteor `forcessl` package back in